packages/jsii-pacmak/lib/targets/kotlin/kdocbuilder.ts

```diff
@@ -0,0 +1,46 @@
+import { CodeMaker } from 'codemaker'
+
+export class KDocBuilder {
+  private readonly chunks: string[] = [];
+
+  public push(
+    chunk: string | undefined,
+    prefix: string | undefined = undefined
+  ) {
+    if (!chunk || chunk.length === 0) {
+      return;
+    }
+
+    if (prefix) {
+      this.chunks.push(`${prefix}${chunk}`);
+      return;
+    }
+
+    this.chunks.push(chunk);
+  }
+
+  private getLines(): string[] {
+    const lines = [];
+    for (const chunk of this.chunks) {
+      if (lines.length > 0) {
+        lines.push('');
+      }
+
+      lines.push(...chunk.split('\n'));
```

Comment:
I wonder if the `split` could (should?) be done upfront. Not that this is bad, just a suggestion.